strax/corrections.py

```diff
@@ -126,3 +143,12 @@ def write(self, correction, df, required_columns=('ONLINE', 'v1')):
         logging.info('Writing')
 
         return df.to_mongo(correction, self.database, if_exists='replace')
+
+    def check_timezone(date):
+        """Smart logic to check date is given in UTC time zone
+        :param date: date e.g. datetime(2020, 8, 12, 21, 4, 32, 7, tzinfo=pytz.utc)
+        """
+        if date.tzinfo == pytz.utc:
+            return date
+        else:
+            raise ValueError(f'{date} must be specify in UTC timezone')
```

Comment:
There is a typo here. Perhaps add that the tz info should be in the datetime object:
```python
ValueError(f'{date} must have UTC timezone. Insert datetime object like "datetime.datetime.now(tz=datetime.timezone.utc)"')```